### PR TITLE
Update the error message for the rabc method

### PR DESCRIPTION
**Pull request by SimFG**
/kind improvement
Signed-off-by: SimFG <bang.fu@zilliz.com>

### CODE REVIEW

File: internal/rootcoord/root_coord.go [hunk 1]
```diff
@@ -2259,7 +2261,8 @@ func (c *Core) isValidRole(entity *milvuspb.RoleEntity) error {
 		return errors.New("the name in the role entity is empty")
 	}
 	if _, err := c.meta.SelectRole(util.DefaultTenant, &milvuspb.RoleEntity{Name: entity.Name}, false); err != nil {
-		return err
+		log.Error("fail to select the role", zap.Error(err))
```
Comment (czs007) [inline internal/rootcoord/root_coord.go:2264]: Warn or info

Comment (SimFG) [reply internal/rootcoord/root_coord.go:2264]: The request will return the err, use warn?

Comment (czs007) [reply internal/rootcoord/root_coord.go:2264]: OK， warn sounds good

Comment (SimFG) [reply internal/rootcoord/root_coord.go]: Have updated

**Pull request merged**